src/viur/core/bones/relational.py

```diff
@@ -1096,7 +1096,7 @@ def get_values(skel_, values_cache):
 
         return result
 
-    def createRelSkelFromKey(self, key: t.Union[str, "db.Key"], rel: dict | None = None):
+    def createRelSkelFromKey(self, key_rel_list: list[tuple]):
```

Comment:
You have now turned the `createRelSkelFromKey` method into `createRelSkelsFromKeys` without supporting the old behavior. Due to the signature change at least, this is a breaking change and belongs in a major version...

If I want to create a relskel without a using skel, I now have to `createRelSkelFromKey([key, None])`. Before, `createRelSkelFromKey(key)` was enough -- That was so much easier.

Either you put it all in an overloaded method, with for example these cases:

- `createRelSkelFromKey(list[db.Key], list[dict | None])`
- `createRelSkelFromKey(list[tuple[db.Key, dict | None]])`
- `createRelSkelFromKey(db.Key, dict | None)`

or separate it:

- `createRelSkelFromKey(db.Key, dict | None)`
- `createRelSkelsFromKeys(list[tuple[db.Key, dict | None]])`

Reply:
Please make sure that a name with plural returns / expect a list. And singular should only deal with one element.